tests/golden/common/common.bats

```diff
@@ -53,11 +53,6 @@ EOF
 
   assert_failure
   assert_output --partial - <<EOF
-Invalid qualified entry path format: "back#/path#\n\nsmth".
-Expected format is: [<backend-name>#]<entry-path>.
-<backend-name> can be a string of the following characters: [a-zA-Z0-9] and symbols '-', '_', ';'.
-Examples: 'vault_kv-backend#secrets/google', 'my/passwords/entry'.
-
```

Comment:
I think we should keep these lines in the test's expected output.

Reply:
done